packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -126,13 +131,15 @@ const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
     }, [history, isMounted, onHistoryChange]);
 
     useEffect(() => {
-        if (reload) {
+        if (reload && !should_disable_livechat) {
```

Comment:
```suggestion
        if (reload || !should_disable_livechat) {
```
Shouldn't it be either then u disable liveChat?